.github/workflows/iroha2-dev-pr.yml

```diff
@@ -278,6 +278,7 @@ jobs:
   bench:
     runs-on: [self-hosted, Linux]
     container: rust:1.52-buster
+    if: ${{ false }}
```

Comment:
Why would you like to disable benchmarks? This way we will not know if can be built or not and also other developers can't see their results in CI.

Reply:
Do we need it on dev prs? We can measure on every commit on dev branch.

Reply:
Sometimes we want to check if performance degraded. Also it is important to check if benchmarks still work.

Reply:
We might add https://github.com/rhysd/github-action-benchmark for alerts.

Reply:
> Sometimes we want to check if performance degraded. Also it is important to check if benchmarks still work.

Sometimes? May be we can trigger them manually then?